salalib/vgamodules/vgavisualglobal.h

```diff
@@ -19,27 +19,24 @@
 #pragma once
 
 #include "salalib/ivga.h"
-#include "salalib/pointdata.h"
 #include "salalib/options.h"
 #include "salalib/pixelref.h"
+#include "salalib/pointdata.h"
 
 #include "genlib/simplematrix.h"
 
-class VGAVisualGlobal : IVGA
-{
-public:
-    std::string getAnalysisName() const override {
-        return "Global Visibility Analysis";
-    }
+class VGAVisualGlobal : IVGA {
+  public:
+    std::string getAnalysisName() const override { return "Global Visibility Analysis"; }
     bool run(Communicator *comm, const Options &options, PointMap &map, bool simple_version) override;
-    void extractUnseen(Node& node, PixelRefVector& pixels, depthmapX::RowMatrix<int>& miscs,
-                       depthmapX::RowMatrix<PixelRef>& extents) {
+    void extractUnseen(Node &node, PixelRefVector &pixels, depthmapX::RowMatrix<int> &miscs,
```

Comment:
If you touch this file - I don't think a chunky function like this should be in the header...